### ISSUE TITLE
Very slow rendering (and hiding) big partial in Google Chrome (but fast in FireFox)

### ISSUE THREAD

**Issue by ukolovda** [hotwired/turbo#1060]
I have partial with `<SELECT>` tag with ~10K options. It render and hide very slow (15 or more seconds on fast computer, several minutes on slow). This happen only on Chrome-family browsers. 
In Mozilla FireFox it worked fast (1 second).

I make screenshots with Performance profile page, please, see attachments.

I think, Turbo can be speed up on `SELECT > OPTION` tags and skip some slow operations.

![Screenshot_20231107_134555](https://github.com/hotwired/turbo/assets/1921594/626d9765-ffa0-4d91-83f0-7acac313ecd5)
![Screenshot_20231107_134331](https://github.com/hotwired/turbo/assets/1921594/616f9a95-490a-4c87-a41e-0f09e6a54a1b)

**Comment by afcapel** [comment]
@ukolovda that call tree shows that the slow part is cloning a node which is part of the browser API, rather than Turbo.

Unfortunately this is a Chrome performance issue, and there's not much Turbo can do about it - we need to clone HTML nodes, there's no way around it! 

We've already reported the bug to Chromium project, hopefully they will fix it at some point.

https://bugs.chromium.org/p/chromium/issues/detail?id=1377830#c14

In case it helps, we found a similar problem in one of our apps. We could workaround the Chrome bug moving some Stimulus controllers out of a form and to another container. The Stimulus controller storing a reference to the form made `cloneNode` slow in Chrome, for some unknown reason.

Another thing that can help is disabling Turbo caching on that particular page, that skips cloning the body to store a cache snapshot.
